packages/react-native-web/src/exports/Modal/ModalFocusTrap.js

```diff
@@ -8,6 +8,7 @@
  * @flow
  */
 
+import type { Node as $IMPORTED_TYPE$_Node } from 'React';
```

Comment:
what is going on here?

Reply:
Dodging use of HTML "Node" already existing